tower_cli/constants.py

```diff
@@ -14,6 +14,8 @@
 
 
 VERSION = '3.3.0'
+# This is the release number for the RPM builds
+RELEASE = 1
```

Comment:
...and the idea here is that code _importing_ tower-cli can deduce what RPM version it is.

Reply:
The RPM version and release should now be controlled by setup.py. I doubt that you will need to change RELEASE often, looks like it might have only happened once in the past that I can tell.

Reply:
> The RPM version and release should now be controlled by setup.py.

It is kind of confusing that this would be true, but it would _also_ be defined in source. It may be less prone to error to just have an overridable default value in the Makefile and leave it as that.

Reply:
Okay, I see what you were saying now.

Reaching out to EPEL on setup seems aggressive to me. I don't want to go that far.

One thing on my mind is tags, for instance. Every initial tag, like `v3.3.2` should have `RELEASE = 1` in constants. If you ever iterated that number, it seems like you would need a tag like `v3.3.2.rev2`. Otherwise, your rpm doesn't technically correspond to a state of the source. But I don't know if that's really a problem, or something we need to care too much about.

Reply:
The other thing you could do is just never change the revision but have a change to the spec file cause a change to the version number. Then we just never have to change from revision 1 but if something is bad in the package we go from 3.3.0 to 3.3.1.

Reply:
True, and I would tend to prefer that. Then ideally, this would be here for the purpose of forever remaining as a static `1`.